plothist/hep_plotters.py

```diff
@@ -427,9 +427,9 @@ def add_luminosity(
     collaboration : str, optional
         Collaboration name, by default "Belle II"
     x : float, optional
-        x position, by default 1.0.
+        x position, by default "right" = 1.0.
```

Comment:
x position in unit of the x-axis length. The default is value "right", which is an alias for 1.0.